src/GeneralStateTestsFiller/stReturnDataTest/tooLongReturnDataCopyFiller.yml

```diff
@@ -0,0 +1,247 @@
+    - data: :label success :abi f(uint,uint,uint,uint) 0x10 0x01 0x0f 0xC0DE
+      accessList: []
+
+    # Receive sixteen bytes, copy sixteen (except for the first one)
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x01 0x10 0xC0DE
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 8-15
+    - data: :label success :abi f(uint,uint,uint,uint) 0x10 0x08 0x08 0xC0DE
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 9-16
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x09 0x08 0xC0DE
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 16-23
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x10 0x08 0xC0DE
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 16-31
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x10 0x10 0xC0DE
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 32-47
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x20 0x10 0xC0DE
+      accessList: []
+
+
+
+    # Show that REVERT behaves the same way as RETURN
+
+    # Receive an empty buffer, copy zero bytes from offset zero.
+    - data: :label success :abi f(uint,uint,uint,uint) 0x00 0x00 0x00 0x0BAD
+      accessList: []
+
+    # Receive an empty buffer, copy the first byte from it
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x00 0x00 0x01 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy the first eight
+    - data: :label success :abi f(uint,uint,uint,uint) 0x10 0x00 0x08 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy the first sixteen
+    - data: :label success :abi f(uint,uint,uint,uint) 0x10 0x00 0x10 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy the first seventeen
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x00 0x11 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy fifteen (except for the first one)
+    - data: :label success :abi f(uint,uint,uint,uint) 0x10 0x01 0x0f 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy sixteen (except for the first one)
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x01 0x10 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 8-15
+    - data: :label success :abi f(uint,uint,uint,uint) 0x10 0x08 0x08 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 9-16
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x09 0x08 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 16-23
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x10 0x08 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 16-31
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x10 0x10 0x0BAD
+      accessList: []
+
+    # Receive sixteen bytes, copy numbers 32-47
+    - data: :label fail    :abi f(uint,uint,uint,uint) 0x10 0x20 0x10 0x0BAD
+      accessList: []
```

Comment:
this creates a lot of empty acessLists
do we need to use eip1559 tr here?

Reply:
No, I just thought we might as well use the latest and greatest. I'll change them to tr type 0.